app/models/user.rb

```diff
@@ -3,24 +3,15 @@
 class User < ApplicationRecord
   PROFILE_IMAGE_EXTENSIONS = %w[image/jpg image/png image/gif].freeze
   MAX_PROFILE_IMAGE_SIZE = 15.megabytes
-  RESUZE_IMAGE_SIZE = [800, 800].freeze
+  RESIZE_IMAGE_SIZE = [800, 800].freeze
 
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_one_attached :profile_image
-  after_save :resize_large_image
-  validates :profile_image, content_type: { in: PROFILE_IMAGE_EXTENSIONS, message: I18n.t('.activerecord.errors.messages.content_type_invalid') },
-                            size: { less_than: MAX_PROFILE_IMAGE_SIZE, message: I18n.t('.activerecord.errors.messages.size_invalid') }
+  validates :profile_image, content_type: { in: PROFILE_IMAGE_EXTENSIONS },
+                            size: { less_than: MAX_PROFILE_IMAGE_SIZE }
```

Comment:
```suggestion
  validates :profile_image, content_type: { in: PROFILE_IMAGE_EXTENSIONS }, size: { less_than: MAX_PROFILE_IMAGE_SIZE }
```

好みの問題ですが1行でもいいかもしれませんね

Reply:
@JunichiIto 
1行にまとめました！